akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedSetup.scala

```diff
@@ -83,5 +86,26 @@ private[persistence] final class EventsourcedSetup[C, E, S](
     this
   }
 
+  private var recoveryTimer: OptionVal[Cancellable] = OptionVal.None
+
+  def startRecoveryTimer(snapshot: Boolean): Unit = {
+    cancelRecoveryTimer()
+    implicit val ec: ExecutionContext = context.executionContext
+    if (snapshot)
+      context.system.scheduler.scheduleOnce(settings.recoveryEventTimeout, context.self.toUntyped,
+        RecoveryTickEvent(snapshot = true))
+    else
+      context.system.scheduler.schedule(settings.recoveryEventTimeout, settings.recoveryEventTimeout,
```

Comment:
this this should be assigned to recoveryTimer

Reply:
oh, fixed, thx